src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -102,17 +106,20 @@
  * zone ID is normalized in the following syntax:
  * <blockquote><pre>
  * <a id="NormalizedCustomID"><i>NormalizedCustomID:</i></a>
- *         {@code GMT} <i>Sign</i> <i>TwoDigitHours</i> {@code :} <i>Minutes</i>
+ *         {@code GMT} <i>Sign</i> <i>TwoDigitHours</i> {@code :} <i>Minutes</i> [<i>Seconds</i>]
```

Comment:
Hello Naoto,

Should this instead be: `... <i>Minutes</i> [{@code :} <i>Seconds</i>]` - i.e. should it have the `:` literal if seconds are present in the custom timezone id?